patches/go-1.22.1/0001-Make-boringcypto-build-compatible-with-aws-lc.patch

```diff
@@ -0,0 +1,60 @@
+From de7e50742be2227a2a87a60dd140ddc8bd4fd4cc Mon Sep 17 00:00:00 2001
```

Comment:
Since this patches will probably be the same for go-1.21.10 I would just make the dir go-1.21.